examples/Client/DistributedLockApi/UnLockApplication/Program.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Threading;
+using Dapr.Client;
+
+namespace DistributedLockApi
+{
+    public class Program2
+    {
+        [Obsolete]
+        public static async Task Main(string[] args)
+        {
+            using var client = new DaprClientBuilder().Build();
+            string StoreName = "redislock";
+            string ResourceId = "resourceId";
+            string LockOwner = "owner2";
+            Int32 ExpiryInSeconds = 25;
+            
+            //Trying to acquire the lock which is being used by the other process(TryLockApplication)
+            var tryLockResponse = await client.TryLock(StoreName, ResourceId, LockOwner, ExpiryInSeconds);
+            Console.WriteLine("Acquired Lock? " + tryLockResponse.Success);
+            Console.WriteLine("Lock cannot be acquired as it belongs to the other process");
+
+            var lockResponse = await client.Unlock(StoreName, ResourceId, LockOwner);
+            Console.WriteLine("Unlock API response when lock is acquired by a different process: " + lockResponse.status);
+
+            System.Threading.Thread.Sleep(25000);
```

Comment:
Instead of a just a `Thread.Sleep` here, can we do a while loop with a sleep where we try and acquire the lock?

I think it's important to show how we think people should be handling the situation of waiting for another app to unlock. 

Not for this PR, but maybe that's something we add as a convenience feature to the client eventually? A blocking lock call?